### PR TITLE
WIP: create an additional comment section to the video form 163

### PR DESCRIPTION
**Pull request by tandaraelena**
Issue #163

### CODE REVIEW

**Review by RadosavAntonio — APPROVED** [review]
💯 for WIP: #163

**Review by BogdanNegura — APPROVED** [review]
💯

**Review by psyhedeliq — APPROVED** [review]
Awesome! 💯 
Thank you Elena! 🥇

**Review by valentin3199 — APPROVED** [review]

**Review by RadosavAntonio — APPROVED** [review]

**Review by valentin3199 — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
Awesome! 💯

**Review by tandaraelena — COMMENTED** [review]
https://www.robinwieruch.de/local-storage-react

**Review by psyhedeliq — APPROVED** [review]

**Review by BogdanNegura — APPROVED** [review]
👍

File: src/component/comment-list/comment-list.component.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { StyledCommentList } from './comment-list.style'
+
+const CommentList = ({commentList}) => {
+  console.log(commentList)
```
Comment (QuintyHH) [inline src/component/comment-list/comment-list.component.jsx]: Woopsies . What is this doing here :D

Comment (marianzburlea) [inline src/component/comment-list/comment-list.component.jsx]: console.log left inside the code

File: src/component/comment-list/comment-list.component.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { StyledCommentList } from './comment-list.style'
+
+const CommentList = ({commentList}) => {
```
Comment (marianzburlea) [inline src/component/comment-list/comment-list.component.jsx]: spaces for `{commentList}` -> `{ commentList }`

File: src/component/comment-list/comment-list.component.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  console.log(commentList)
+  return ( commentList === null ?
+    '' :
```
Comment (marianzburlea) [inline src/component/comment-list/comment-list.component.jsx:8]: A `No comments` message would look better and more informative.

**Review by marianzburlea — CHANGES_REQUESTED** [review]
It's not connected to the database

**Review by valentin3199 — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**